libbpf-tools/tcprtt.bpf.c

```diff
@@ -29,8 +29,7 @@ struct {
 
 static struct hist zero;
 
-SEC("fentry/tcp_rcv_established")
-int BPF_PROG(tcp_rcv, struct sock *sk)
+static __always_inline int handle_tcp_rcv_established(struct sock *sk)
```

Comment:
`__always_inline` is not necessary.

Reply:
We hit some issues with this in Inspektor Gadget: https://github.com/inspektor-gadget/inspektor-gadget/pull/1463. I can remove that here if you think it's not an issue for this case.

Reply:
According to this comment, we should not hit this issue in tracing progs.

https://github.com/torvalds/linux/blob/219d54332a09e8d8741c1e1982f5eae56099de85/kernel/bpf/verifier.c#L6004-L6007

Reply:
Thanks for the link. I removed it.